vespalib/src/vespa/vespalib/util/generation_hold_list.h

```diff
@@ -0,0 +1,83 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "array.h"
```

Comment:
Consider including vector instead of array.h

Reply:
Fixed.